src/index.html

```diff
@@ -11,7 +11,136 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+    <form action="http://localhost:3000/api" method="post">
+      <fieldset class="gr_form">
+        <legend>Personal information</legend>
+        <label class="form">
+          Surname:
+          <input name="surname"
```

Comment:
<img width="1078" alt="Screenshot 2023-01-10 at 17 51 23" src="https://user-images.githubusercontent.com/53228259/211598274-9c4c40db-c784-42ce-88b0-4a57c3bff178.png">

Reply:
Fixed the errors